.golangci.yml

```diff
@@ -0,0 +1,34 @@
+run:
+  deadline: 10m
+
+linters:
+  enable:
+    - bodyclose
+    - gofmt
+    - goimports
+    - revive
+    - gosec
+    - misspell
+    - unconvert
+    - unparam
+    - goheader
+    - gocritic
+
+# TODO(lianghao208): enable misspell
```

Comment:
looks like misspell is already enabled, why is this comment needed

Reply:
done, already removed the comment.